### PR TITLE
[ACM-10578] - partial match label

### PR DESCRIPTION
**Pull request by SherinV**
### Related Issue

https://issues.redhat.com/browse/ACM-10578

### Description of changes
- Enabled partial match for array and object properties.

### CODE REVIEW

File: pkg/resolver/searchHelper.go
Comment (jlpadilla) [reply pkg/resolver/searchHelper.go:61]: Nevermind, I see WIP.

**Comment by SherinV** [comment]
@jlpadilla Need final review on this, when you get a chance. Thanks.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

**Review by jlpadilla — APPROVED** [review]
/lgtm

**Pull request merged**
